TrafficCapture/dockerSolution/src/main/docker/migrationConsole/kafkaExport.sh

```diff
@@ -55,25 +59,29 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
-partition_offsets=$(./kafka-tools/kafka/bin/kafka-run-class.sh kafka.tools.GetOffsetShell --broker-list "$broker_endpoints" --topic "$topic" --time -1 $(echo "$kafka_command_settings"))
+partition_offsets=$(./kafka/bin/kafka-run-class.sh kafka.tools.GetOffsetShell --broker-list "$broker_endpoints" --topic "$topic" --time -1 $(echo "$kafka_command_settings"))
 comma_sep_partition_offsets=$(echo $partition_offsets | sed 's/ /,/g')
 echo "Collected offsets from current Kafka topic: "
 echo $comma_sep_partition_offsets
 
 epoch_ts=$(date +%s)
-file_name="kafka_export_$epoch_ts.proto"
-archive_name="$file_name.tar.gz"
-group="export_$epoch_ts"
+dir_name="kafka_export_$epoch_ts"
+mkdir -p $dir_name
+archive_name="kafka_export_from_migration_console_$epoch_ts.tar.gz"
+group="exportFromMigrationConsole_$(hostname -s)_$$_$epoch_ts"
+echo "Group name: $group"
 
 set -o xtrace
-./runJavaWithClasspath.sh org.opensearch.migrations.replay.KafkaPrinter --kafka-traffic-brokers "$broker_endpoints" --kafka-traffic-topic "$topic" --kafka-traffic-group-id "$group" $(echo "$msk_auth_settings") --timeout-seconds "$timeout_seconds" --partition-limit "$comma_sep_partition_offsets" >> "$file_name"
+./runJavaWithClasspath.sh org.opensearch.migrations.replay.KafkaPrinter --kafka-traffic-brokers "$broker_endpoints" --kafka-traffic-topic "$topic" --kafka-traffic-group-id "$group" $(echo "$msk_auth_settings") --timeout-seconds "$timeout_seconds" --partition-limits "$comma_sep_partition_offsets" --output-directory "./$dir_name"
 set +o xtrace
 
-tar -czvf "$archive_name" "$file_name" && rm "$file_name"
+cd $dir_name
+tar -czvf "$archive_name" *.proto && rm *.proto
```

Comment:
I'm fine doing this later, but this whole script would be much faster and easier to manage if you wrote compressed streams first, then tarred them together.  As it is, you'll have a huge number of bytes going to disk (which might be remote) and back.  Since the java program uses multiple files, you'd have to manage it within the java program (with a GZipInputStream).